README.md

```diff
@@ -1083,8 +1083,7 @@ This verification directive applies to HTTP/1 transactions. For HTTP/2, status
 verification is specified via `:status` pseudo header field verification using
 the field verification mechanism described above.
 
-Verification of HTTP response reason strings, such as "Not Found", is not
-currently supported.
+Verification of HTTP response reason strings, such as "Not Found", is also supported.
```

Comment:
PR looks great. Let's expand the documentation just a bit. Can you:

1. Move this sentence to the end of the first paragraph in this section. Let's tweak it to something like: `Both response code, such as "403", and HTTP/1 response reason string, such as "Forbidden", are supported for verification.
2. Add reason string example to the `proxy-response:` node example in this section. `reason: Not Found`.

Reply:
Thanks for reviewing this. Updated the doc per your suggestions.